hax/__init__.py

```diff
@@ -63,6 +72,19 @@ def init(filename=None, **kwargs):
     # Override with kwargs
     config.update(kwargs)
 
+    # Load unblinding selection
+    unblinding_filename = os.path.join(hax_dir, 'unblinding.ini')
+
+    global unblinding_config
+    unblinding_configp = ConfigParser(inline_comment_prefixes='#', strict=True)
+    unblinding_configp.read(unblinding_filename)
+
+    # Evaluate the values in the ini file
+    unblinding_config = {}
+    section_to_use = 'UNBLINDING'
+    for key, value in unblinding_configp[section_to_use].items():
+        unblinding_config[key] = ast.literal_eval(value)
```

Comment:
Since you're only storing two settings  in unblinding.ini, and the unblinding.ini filename can't be overriden by the user, would it be easier to make unblinding.ini just unblinding.py? Then add unblinding here https://github.com/XENON1T/hax/blob/master/hax/__init__.py#L7, and you can access the settings as `hax.unblinding.unblinding_selection` and `hax.unblinding.blind_from_run` when you need them in hax.runs.

Reply:
it will soon have many entries after #199 [here](https://github.com/XENON1T/hax/pull/199/files#diff-f5b19e12c8f52ab242b67b1d087318a1R114). but i guess your suggestion applies there too, and would clean up the ```init``` function, right?

Reply:
Yes, it should work just as well. I haven't looked at #199 in detail, but maybe you want to put all settings in a `settings` dictionary inside `unblinding.py` if you have to do a lot of dynamic key access (`getattr(hax.unblinding, key)` isn't so great). You could even move `is_blind` into the file as well to have all blinding logic together.

Reply:
Great suggestion! Implemented in 1cc8c65 and 309112f